chapters/stress/index.md

```diff
@@ -0,0 +1,271 @@
+
+Note that in these tests, the quality of the results is not assessed, and therefore only the amount of returned matches is validated.
+
+### Image search
+
+The first scenario simulates an image search application. Images are randomly generated and indexed using a Flow with this topology:
+
+![Image Index Flow](image_index_flow.png)
+
+We use the `mobilenet_v2` `Encoder` with the `ImageTorchEncoder`.
+
+At query time, the Flow looks like this:
+
+![Image Query Flow](image_query_flow.png)
+
+In the image search scenario, there is no `Segmentation` and ranking. The results are returned according to the `distance` in embedding space returned by the `VectorIndexers`. 
+
+### Text search
+
+The text search topology is similar to the one above, but with a `Segmenter` (`Sentencizer`) and a `Ranker` (`SimpleAggregateRanker`). This means we are indexing on the chunk level of a `Document`. 
+
+The full indexing Flow can be seen below:
+
+![Text Index Flow](text_index_flow.png)
+
+The full search Flow can be seen below:
+
+![Text Query Flow](text_query_flow.png)
+
+## Configuration
+
+The configuration parameters of this experiment can be found in the `.env` file, whose values are set as environment variables and changed in the `flow` and `pod` yamls.
+
+The parameters are organized in 4 groups:
+
+- Infrastructure parameters: Configure what the different machines are.
+- Sharding Parameters: Configure the parallelization and sharding of different components
+- Functional/Indexers parameters: Configure the functional parameters that can affect the performance of the search
+- Client parameters: Configure how the client connects to the Flow
+
+### Sharding and scheduling parameters
+
+Jina allows for parallelization of data processing. You can read an overview of these options [here](https://docs.jina.ai/chapters/parallel/index.html). We use these to parallelize the various `Pods`. On the same topic, we also configure the `scheduling` strategy.
+
+### Functional/Indexers parameter
+
+As part of the benchmark, we also compare the performance of the `NumpyIndexer` with the advanced indexers, based on `Annoy` and `Faiss`. We also provide specific arguments to these: `num_tress` for `Annoy`; `index_key` for `Faiss`.
+
+### Client  
+
+In order to simulate a real-world scenarios, we also test using multiple concurrent clients. This is achieved using the Python `multiprocessing` library.
+
+In order to stress-test the system, we also issue index and query requests for several hours in a row. Each request contains a specific number of `Documents`, split across requests of the same `request_size`.
+
+The client also sets the `TOP_K` parameter. This limits the number of matches returned by the Flow. This also affects performance.
+
+## Experiments
+
+**NOTE** For now we only provide results for the `image` dataset. The `text` dataset will work too, if you want to test that.
```

Comment:
Do we need backticks here?

Reply:
It refers to the folder names. Check again